pkgs/by-name/go/gokey/package.nix

```
@@ -1,16 +1,18 @@
 { lib, buildGoModule, fetchFromGitHub }:
 buildGoModule {
```

```suggestion
buildGoModule rec {
```

Missing a rec here, and please squash the commits.

---

Done. Thanks for the review.